draft-ietf-rats-eat.md

```diff
@@ -1198,104 +1150,40 @@ The string identifying the JSON-encoded token MUST be one of the following:
-For example, one register may be writable only by the TEE, so the detached claims from the TEE will have TEE-level security.
+A nested token need not use the same encoding as the enclosing token.
+This enables composite devices to be built without regards to the encoding used by components.
+Thus, a CBOR-encoded EAT can have a JSON-encoded EAT as a nested token and vice versa.
 
-The data type for this type of submodule MUST be an array
-It contains two data items, a hash algorithm identifier and a byte string containing the digest.
+#### Detached Submodule Digest
 
-The hash algorithm identifier is always from the COSE Algorithm registry, {{IANA.COSE.Algorithms}}.
-Either the integer or string identifier may be used.
-The hash algorithm identifier is never from the JOSE Algorithm registry.
+The Detached-Submodule-Digest type is similar to a Claims-Set submodule, except a digest of the Claims-Set is included in the claim with the Claims-Set contents conveyed separately. The separately conveyed Claims-Set is called a detached claims set. The input to the digest algorithm is the byte-string wrapped encoded form of the detached Claims-Set.
 
-When decoding a CBOR format token, the detached digest type is distinguished from the other types by it being an array.
-In CBOR encoded tokens none of other submodule types are arrays.
+The data type for this type of submodule is an array consisting of two data items: an algorithm identifier and a byte string containing the digest. The hash algorithm identifier is always from the COSE Algorithm registry, {{IANA.COSE.Algorithms}}. Either the integer or string identifier may be used. The hash algorithm identifier is never from the JOSE Algorithm registry.
 
-When decoding a JSON format token, a little more work is required because both the nested token and detached digest types are an array.
-To distinguish the nested token from the detached digest, the first element in the array is examined.
-If it is "JWT" or "BUNDLE", then the submodule is a nested token.
-Otherwise it will contain an algorithm identifier and is a detached digest.
-
-A detached EAT bundle, described in {{DEB}}, may be used to convey detached claims sets and the token with their detached digests.
-EAT, however, doesn't require use of a detached EAT bundle.
-Any other protocols may be used to convey detached claims sets and the token with their detached digests.
+A detached EAT bundle, described in {{DEB}}, may be used to convey detached claims sets and the EAT containing the corresponding detached digests.
+The EAT format, however, doesn't require use of a detached EAT bundle.
+Any other protocols may be used to convey detached claims sets and the EAT containing the corresponding detached digests.
 Note that since detached Claims-Sets are signed, protocols conveying them must make sure they are not modified in transit.
```

Comment:
Replace with "Detached Claims-Sets must not be modified in transit. If they are the digest won't validate them." ?

Or delete?

The existing text sounds to much like the protocol conveying them must provide integrity protection and has thus confused some readers.

Reply:
Changed the sentence beginning with "Note that since" per your suggestion.